contrib/autocomplete/bash_autocomplete

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+: ${PROG:=$(basename ${BASH_SOURCE})}
```

Comment:
Given that we're now copying this file; should we consider renaming it to `/contrib/completion/bash/ctr`, and update the script so that it no longer requires the `PROG` variable to be set?